src/deepin-system-upgrade-tool/src/tool/upgrade_progress/systemupgradewidget.cpp

```diff
@@ -50,8 +51,15 @@ void SystemUpgradeWidget::initConnections()
     });
     connect(m_upgradeWidget, &UpgradeWidget::done, this, &SystemUpgradeWidget::jumpToBackupWidget);
     connect(m_backupWidget, &BackupWidget::done, [this] {
-        // Start upgrade when backup is done.
-        emit m_dbusWorker->StartUpgrade();
+        AlertDialog dlg(this);
+        if (dlg.execWarningDialog(nullptr, tr("After clicking restart, the system will be upgraded, please save the current running application data before confirming"), tr("restart now")) == 0)
```

Comment:
restart now 是按钮上的文字的话，首字母应该大写的吧（如果改的话，翻译文件里的应该也要更新）
```suggestion
        if (dlg.execWarningDialog(nullptr, tr("After clicking restart, the system will be upgraded, please save the current running application data before confirming"), tr("Restart now")) == 0)
```

Reply:
down

Reply:
done